modules/rosa-policy-change-management.adoc

```diff
@@ -66,3 +68,161 @@ Because the required permissions can change between y-stream releases, the polic
+
+- Monitor and manage the deployment of Amazon EC2 control plane and infrastructure nodes on the cluster.
+
+|- Monitor and manage Amazon EC2 worker nodes by creating a
+machine pool using the OpenShift Cluster Manager or the ROSA CLI (`rosa`).
+- Manage changes to customer-deployed applications and application data.
+
+|Cluster version
+|**Red Hat**
+
+- Enable upgrade scheduling process.
+
+- Monitor upgrade progress and remedy any issues encountered.
+
+- Publish change logs and release notes for minor and maintenance upgrades.
+
+|- Schedule maintenance version upgrades either immediately, for the future, or have automatic upgrades.
+- Acknowledge and schedule minor version upgrades.
+- Ensure the cluster version stays on a supported minor version.
+- Test customer applications on minor and maintenance versions to ensure compatibility.
+
+|Capacity management
+|**Red Hat**
+
+- Monitor the use of the control plane. Control planes include control plane nodes and infrastructure nodes.
+
+- Scale and resize control plane nodes to maintain quality of service.
+
+| - Monitor worker node utilization and, if appropriate, enables the auto-scaling feature.
+- Determine the scaling strategy of the cluster. See the additional resources for more information on machine pools.
+- Use the provided {cluster-manager} controls to add or remove additional worker nodes as required.
+- Respond to Red Hat notifications regarding cluster resource requirements.
+
+|Virtual storage management
+|**Red Hat**
+
+- Set up and configure Amazon EBS to provision local node storage and persistent volume storage for the cluster.
+
+- Set up and configure the built-in image registry to use Amazon S3 bucket storage.
+
+- Regularly prune image registry resources in
+Amazon S3 to optimize Amazon S3 usage and cluster performance.
+
+| - Optionally configure the Amazon EBS CSI driver or the Amazon
+EFS CSI driver to provision persistent volumes on the cluster.
+
+|AWS software (public AWS services)
+|**AWS**
+
+**Compute:** Provide the Amazon EC2 service, used for
+ROSA control plane, infrastructure, and worker nodes.
+
+**Storage:** Provide Amazon EBS to allow the ROSA service to provision local node storage and persistent volume storage for the cluster.
+
+**Storage:** Provide Amazon S3 for the ROSA service’s
+built-in image registry.
+
+**Networking:**
+Provide the following AWS Cloud services
+to satisfy ROSA virtual networking
+infrastructure needs:
+
+** Amazon VPC
+** Elastic Load Balancing
+** AWS IAM
+
+**Networking:**
+Provide the following optional AWS service integrations for ROSA:
+
+- AWS VPN
+- AWS Direct Connect
+- AWS PrivateLink
+- AWS Transit Gateway
+
+| - Sign requests using an access key ID and secret access key
+associated with an IAM principal or STS temporary security
+credentials.
+- Specify VPC subnets for the cluster to use during cluster
+creation.
+- Optionally configure a customer-managed VPC for use with ROSA clusters.
```

Comment:
We might want to note that this is a requirement for PrivateLink and HCP clusters.

Reply:
agreed